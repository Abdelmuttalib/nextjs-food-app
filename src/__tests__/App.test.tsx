import { screen, render } from "@testing-library/react";
import "@testing-library/jest-dom";
import App from "../App";

describe("App", () => {
  it("renders App component", () => {
    render(<App />);
  });

  it("render Hello World heading", () => {
    render(<App />);

    const heading = screen.getByRole("heading", {
      name: /hello world/i,
    });

    expect(heading).toBeInTheDocument();
  });

  it("renders App component unchanged", () => {
    const { container } = render(<App />);

    expect(container).toMatchSnapshot();
  });
});
