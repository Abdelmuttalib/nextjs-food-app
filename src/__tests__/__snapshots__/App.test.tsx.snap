// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders App component unchanged 1`] = `
<div>
  <div>
    <h1>
      Hello World
    </h1>
  </div>
</div>
`;
